File: launchdarkly/csharp/src/OSEG.LaunchDarklyExamples/PostApprovalRequestApplyExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Org.LaunchDarklyTools.Api;
using Org.LaunchDarklyTools.Client;
using Org.LaunchDarklyTools.Model;

namespace OSEG.LaunchDarklyExamples;

public class PostApprovalRequestApplyExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.ApiKey = new Dictionary<string, string> {["ApiKey"] = "YOUR_API_KEY"};

        var postApprovalRequestApplyRequest = new PostApprovalRequestApplyRequest(
            comment: "Looks good, thanks for updating"
        );

        try
        {
            var response = new ApprovalsApi(config).PostApprovalRequestApply(
                id: null,
                postApprovalRequestApplyRequest: postApprovalRequestApplyRequest
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling ApprovalsApi#PostApprovalRequestApply: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: launchdarkly/typescript-axios/src/PostApprovalRequestApplyExample.ts
import axios, { AxiosError } from "axios";
import * as api from "launchdarkly_client"

const configuration = new api.Configuration({
  apiKey: "YOUR_API_KEY",
});

const postApprovalRequestApplyRequest: api.PostApprovalRequestApplyRequest = {
  comment: "Looks good, thanks for updating",
};

new api.ApprovalsApi(configuration).postApprovalRequestApply(
  "id_string", // id
  postApprovalRequestApplyRequest,
).then(response => {
  console.log(response.data);
}).catch((error: Error | AxiosError) => {
  console.log("Exception when calling ApprovalsApi#postApprovalRequestApply:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});
